ApiGateway/Program.cs

```diff
@@ -20,9 +26,20 @@
 builder.Services.AddSingleton<Cache<Signal>>();
 builder.Services.AddSingleton<Cache<Tracksection>>();
 
-// TODO: Connection details should be loaded from a config provider
 builder.Services
-    .AddSingleton(static services => new MQTTService(services.GetRequiredService<ILogger<MQTTService>>(), "webapi", "192.168.255.6", 1883, "richard", "password", "signalling/", true, true))
+    .AddSingleton(
```

Comment:
Put the defaults into `appsettings.json`, and use environment variables or an `appsettings.Development.Json` for overrides. the Lines 34-38 then become `builder.Configuration.GetValue<string>("MQTT_HOST")` and similar

Also may want to consider an `MQTT` section inside the config (`var MQTTConfiguration = builder.Configuration.GetSection("MQTT");`)

Reply:
Will do in the future, just not in this instance - this merge is mostly to get the audiosystem project in.